customer_credit_limit/security/check_credit_limit_security.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+	<data>
+
+		<record id="group_so_credit_block_releaser" model="res.groups">
+        	<field name="name">Sales Order Credit Block Releaser</field>
+			<field name="category_id" ref="base.module_category_sales_management"/>
```

Comment:
This will break the rendering of the original sales groups in the dropdown. Please give it another category, or (my personal preference) assign the privileges to the sales manager instead of adding a dedicated group for this.